{"name": "登录异常，测试数据是:{'accounts': 'yaoyao', 'pwd': '123456', 'exp': '密码错误'}", "status": "passed", "description": "\n        对登录接口进行异常登录的测试\n        :param args: 参数化的数据\n        :return:\n        ", "attachments": [{"name": "log", "source": "d370f613-77d5-4530-8195-822f7dac72b2-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "args", "value": "{'accounts': 'yaoyao', 'pwd': '123456', 'exp': '密码错误'}"}], "start": 1634269421791, "stop": 1634269421912, "uuid": "262f6259-12a7-42cf-87a2-5445f5cd29fd", "historyId": "9b1c39a42a17c4eb61b2e9ff0cabd0ab", "testCaseId": "acfd50c72596b11b4392c40d7652bccf", "fullName": "script.test_login.TestLogin#test_login_error", "labels": [{"name": "story", "value": "异常数据的登录接口测试"}, {"name": "parentSuite", "value": "script"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "LAPTOP-95KM4O2I"}, {"name": "thread", "value": "32616-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "script.test_login"}]}